server2/Arcadia.Assistant/Arcadia.Assistant.NotificationTemplates/DictionaryExtensions.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace Arcadia.Assistant.NotificationTemplates
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+    using System.Runtime.Serialization;
+
+    public static class DictionaryExtension
+    {
+        public static Dictionary<T1, T2> Perform<T1, T2>(
+            IEnumerable<KeyValuePair<T1, T2>> target,
+            IEnumerable<KeyValuePair<T1, T2>> source)
+        {
+            return target
+                .Concat(source)
+                .GroupBy(x => x.Key)
+                .ToDictionary(x => x.Key, x => x.Last().Value);
+        }
+
+        public static Dictionary<string, string> SerializeObject(object obj)
+        {
+            return new Dictionary<string, string>()
+                .SerializeObject(obj);
+        }
+
+        public static Dictionary<string, string> SerializeObject(
```

Comment:
no way serialization should be an extension to dictionary class - a separate serialize/deserialize class should be used

Reply:
This code copy/pasted from first version.